tests/run-tests

```diff
@@ -552,6 +552,8 @@ the last matching regex is used:
   run-tests -i async - exclude all, then include tests containg "async" anywhere
   run-tests -e '/big.+int' - include all, then exclude by regex
   run-tests -e async -i async_foo - include all, exclude async, yet still include async_foo
+Expectations for tests that do not work in CPython can be saved in a .exp file. That file can be generated by
+running the test without .exp file and saving the .out file produced as a result of the failing test.
```

Comment:
This paragraph is much better suited to be in `tests/README` which describes how to write tests.

Reply:
I disagree with the absolute tone of your comment: I would not have seen it in that document because I did not see that document at all, so it would not have better there 'cause I would not have found it.
Whatever...

Reply:
Ok, I take the point that the tool itself could better describe what it does and how it does it.